web/templates/blocks/health_container.html.tmpl

```diff
@@ -1,15 +1,15 @@
 {{ define "health_container" }}
     <h5>Health</h5>
     <div class="health-container{{ if eq .Layout "horizontal" }} horizontal-container{{ end }}">
-        <div class="alert alert-inline alert-success">
+        <div class="alert alert-inline alert-success health-passing">
```

Comment:
Wasn't it possible to re-use `alert-success` for this matter? I'm guessing it's a selector for the tests, right? (same goes for above ones)

Reply:
I didn't want to couple with actual styling :smile:

Reply:
Fair point, I guess these classes could change after updating to a new EOS design version or whatever. Fine by me then.